scimath/units/electromagnetism.py

```diff
@@ -23,19 +30,25 @@
 ###############################################################################
 
 volts = volt
-v = volt
```

Comment:
Can you keep existing definitions - removing them will break existing code that expects them.